apps/09todo-jquery/script.js

```diff
@@ -1,5 +1,46 @@
 'use strict';
 
 $(document).ready(function() {
-  // App logic goes here
+    // App logic goes here
+    $('form').submit(function(event) {
+        // this only sends the info we want not whatever the web page wants
+        event.preventDefault();
+
+        // this finds the value of the id "todo", and saves it in  a variable.
+        var todoText = $(this).find('#todo').val();
+
+        // variables for the open circle and closed circle.
+        var checkMark = "<i class='check fa fa-circle' aria-hidden='true'></i>";
+        var noCheckMark = "<i class='check fa fa-circle-thin' aria-hidden='true'></i>"
+        var mark = noCheckMark;
+
+
+        // .sortable() allows the user to re-sort the unordered list.
+        // this adds the li into the unordered list.
+        // you can add full html using jquery by putting it into a string.
+        $("#todo-list").append("<li><i class='check fa fa-circle-thin' aria-hidden='true'></i>" + todoText + "<i class='button fa fa-times' aria-hidden='true'></i></li>").sortable();
```

Comment:
Snazzy work here on this app. I would break up your append logic using string concatenation so its easily make and track changes like `fa fa-circle`.